zcash_primitives/src/transaction/components/orchard.rs

```diff
@@ -185,7 +185,7 @@ pub fn write_v5_bundle<W: Write>(
 
         write_flags(&mut writer, &bundle.flags())?;
```

Comment:
Can this be replaced by the new `orchard` APIs? (And similarly for reading.)

Reply:
Serialization has not been moved to the `orchard` crate, just the construction of the txid and auth hashes.

Reply:
Oh, I guess though that `Flags::to_byte` has. Will do.